pkg/virtctl/vm/vm.go

```diff
@@ -38,18 +38,21 @@ import (
 )
 
 const (
-	COMMAND_START          = "start"
-	COMMAND_STOP           = "stop"
-	COMMAND_RESTART        = "restart"
-	COMMAND_MIGRATE        = "migrate"
-	COMMAND_MIGRATE_CANCEL = "migrate-cancel"
-	COMMAND_GUESTOSINFO    = "guestosinfo"
-	COMMAND_USERLIST       = "userlist"
-	COMMAND_FSLIST         = "fslist"
-	COMMAND_ADDVOLUME      = "addvolume"
-	COMMAND_REMOVEVOLUME   = "removevolume"
+	COMMAND_START             = "start"
+	COMMAND_STOP              = "stop"
+	COMMAND_RESTART           = "restart"
+	COMMAND_MIGRATE           = "migrate"
+	COMMAND_MIGRATE_CANCEL    = "migrate-cancel"
+	COMMAND_GUESTOSINFO       = "guestosinfo"
+	COMMAND_USERLIST          = "userlist"
+	COMMAND_FSLIST            = "fslist"
+	COMMAND_MEMORYDUMP        = "memory-dump"
+	COMMAND_REMOVE_MEMORYDUMP = "remove-memory-dump"
```

Comment:
rather than a `remove-memory-dump` command, would `memory-dump create` and `memory-dump delete` be better?

Reply:
its not quite delete since you dont delete the memory dump you remove it from the vm but it still exists